Makefile

```diff
@@ -56,34 +56,25 @@ endif
 #
 # Catch-all rules
 #
+.DEFAULT_GOAL := help
+
+# start with displaying help
+help:
```

Comment:
Duplicated target **help**. See https://github.com/owncloud/user_ldap/pull/402/files#diff-b67911656ef5d18c4ae36cb6741b7965L65

Reply:
Thanks for noticing! Fix in #404 https://github.com/owncloud/user_ldap/pull/404/commits/b981f4fc2959aa949263df9ab87f7614bca19e8d